src/main/java/com/dbteam/utils/DoubleHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.dbteam.utils;
+
+import org.springframework.context.annotation.Bean;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class DoubleHelper {
```

Comment:
```suggestion
@Bean
public class DoubleHelper {
```

Reply:
@Bean annotation is not applicable to classes. I think we can leave this class without any spring boot annotation as it has and will have in future only public static methods ;)